etl/steps/data/meadow/unep/2023-01-03/renewable_energy_investments.py

```diff
@@ -0,0 +1,53 @@
+"""Extract data from UNEP's Global trends in renewable energy investment.
+
+Since the data is given as an image of a table (not suitable for OCR) the data has been manually copied into a file,
+next to this script.
+"""
+
+import pandas as pd
+from owid.catalog import Dataset, Table
+
+from etl.helpers import PathFinder
+from etl.steps.data.converters import convert_snapshot_metadata
+
+# naming conventions
+paths = PathFinder(__file__)
+
+EXTRACTED_DATA_FILE = paths.directory / "extracted_data.csv"
```

Comment:
If you ever change the data, the step won't be rebuilt because it doesn't use `extracted_data.csv` to compute checksum. It has to have prefix `renewable_energy_investments*` or you have to put both files to `renewable_energy_investments/` folder.

Reply:
Thanks. I tried calling it `renewable_energy_investments_data.csv`, but that was not recognised. However, `renewable_energy_investments.data.csv` does work. So it has to be `name_of_step.something`, right?